### PR TITLE
Bump: Sentry-Android to 5.6.1 and Sentry-Cocoa to 7.9.0

### PR DESCRIPTION
**Pull request by dgilperez**
## :scroll: Description

Bump: Sentry-Android to 5.6.1 and Sentry-Cocoa to 7.9.0

## :bulb: Motivation and Context

Changes upstream seem fixes and non breaking additions, it would be nice to use some of those errors in the dart version.

## :green_heart: How did you test it?

Run the test suite locally, green.

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] I updated the docs if needed
- [X] All tests passing
- [X] No breaking changes

## :crystal_ball: Next steps

Grab a 🍺 😂

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 * Feat: Support maxSpan for performance API and expose SentryOptions through Hub (#716)
 * Fix: await ZonedGuard integration to run (#732)
 * Fix: `sentry_logging` incorrectly setting SDK name (#725)
+* Bump: Sentry-Android to 5.6.1 and Sentry-Cocoa to 7.9.0 (#735)
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
* Bump: Sentry-Android to 5.6.1 and Sentry-Cocoa to 7.9.0 (#736)
```

**Comment by marandaneto** [comment]
@dgilperez thanks for doing this.
Please update your main branch, seems to be outdated.
Also, fix descriptions to match the versions, looks like its a copy-paste error.

**Comment by dgilperez** [comment]
Fixed. You're in deed fast :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,278 +1,279 @@
 # Unreleased
 
-* Feat: Support maxSpan for performance API and expose SentryOptions through Hub (#716)
-* Fix: await ZonedGuard integration to run (#732)
-* Fix: `sentry_logging` incorrectly setting SDK name (#725)
+- Feat: Support maxSpan for performance API and expose SentryOptions through Hub (#716)
```
Comment (marandaneto) [inline CHANGELOG.md]: Roll back the changes from `*` to `-` since this is the already defined way.

Comment (dgilperez) [reply CHANGELOG.md]: I am sorry for that, my linter played me :(

**Review by marandaneto — APPROVED** [review]

**Comment by marandaneto** [comment]
thanks @dgilperez :)

**Comment by dgilperez** [comment]
You're welcome, and super fast :)

**Comment by marandaneto** [comment]
> You're welcome, and super fast :)

Keep 'em coming. :)

**Pull request merged**